proxies/live/apiproxy/resources/jsc/RestrictRequest.js

```diff
@@ -3,12 +3,14 @@ var request_pathsuffix = context.getVariable('proxy.pathsuffix');
 var request_path_nhs_number = request_pathsuffix.split("/")[2];
 var method_is_restricted = true
 var sync_wrapped = context.getVariable('request.header.x-sync-wrapped');
+var vot = context.getVariable('jwt.DecodeJWT.DecodeIdToken.claim.vot');
+var allowed_vots = ["P9.Cp.Cd","P9.Cm","P9.Cp.Ck"];
 
```

Comment:
my js brain wants this to be a const.

Reply:
that's what I originally tried, as per ecma 2015, but 'const' was not recognised on parsing and caused errors, which I was very surprised at